db/reset_db.js

```diff
@@ -1,7 +1,19 @@
 const { User, FrequentProblem } = require('../models')
 
 User.sync({force: true})
-FrequentProblem.sync({force: true})
+FrequentProblem.sync({force: true}).then(function () {
+
+    FrequentProblem.destroy({truncate: true})
+        .then( () => frequentProblemValues.forEach( ([env, subEnv, subject, solution, solutionURL]) => {
+            FrequentProblem.create({env, subEnv, subject, solutionURL }).then(function(user) {
+                // you can now access the newly created task via the variable task
+                console.log('success');
+            }).catch(function(err) {
+                // print the error details
+                console.log(err, env, subEnv, subject, solutionURL );
```

Comment:
I copied from a stackoverflow solution so we can see where the error occurred, so i'm not sure. . I can remove it for now.